terraform/aks/variables.tf

```diff
@@ -71,6 +71,26 @@ variable "review_url_db_name" {
   default     = null
   description = "the name of the secret storing review db url"
 }
+
+variable "review_db_dbname" {
+  default     = null
+  description = "the name of the secret storing review db name"
```

Comment:
Could you please start with a upper case letter?

Reply:
done